src/content/docs/pt-br/guides/authentication.mdx

```diff
@@ -0,0 +1,189 @@
+---
+title: Autenticação
+description: Uma introdução a autenticação no Astro
+i18nReady: true
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import ReadMore from '~/components/ReadMore.astro'
+
+Autenticação e autorização são dois processos de segurança que te ajudam a controlar quem possui acesso a diferentes parte do seu website ou aplicativo. Autenticação é o processo de verificar a identidade de um visitante, e autorização é o processo de permitir aos visitantes acesso a recursos e rotas protegidas.
+
+Autenticação te permite customizar áreas do seu site para indivíduos logados e fornece proteção máxima a informações pessoais ou privadas. Bibliotecas de autenticação  (e.g. [Lucia Auth](https://lucia-auth.com/), [Auth.js](https://authjs.dev/)) fornecem utilidades para múltiplos métodos de autenticação como acesso por email e provedores OAuth.
+
+:::tip
+Oficialmente, não existe solução para autenticação no Astro, mas você pode encontrar [integrações "auth" da comunidade](https://astro.build/integrations/?search=auth) no diretório de integrações.
+:::
+
+<ReadMore>Veja como [adicionar autenticação com Supabase](/en/guides/backend/supabase/#adding-authentication-with-supabase) ou [adicionar autenticação com Firebase](/en/guides/backend/google-firebase/#adding-authentication-with-firebase) em nossos guias dedicados a esses serviços backend.</ReadMore>
+
+## Lucia
+
+Lucia é biblioteca de autenticação baseada em sessões que não depende de frameworks que possui grande suporte para Astro.
+
+### Instalação
+
+Instale Lucia utilizando o gerenciador de pacotes da sua escolha.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install lucia
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm add lucia
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add lucia
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+### Configuração
+
+Use [o guia "Getting started in Astro"](https://lucia-auth.com/getting-started/astro) para inicializar a Lucia com um adaptador e implementar um banco de dados para armazenar usuários e sessões.
```

Comment:
We translate titles also, if you want you can add a note to warn the user that the content is in English. We have some other examples, I think that in the community recipes.
```suggestion
Use [o guia "Começando com Astro" da Lucia](https://lucia-auth.com/getting-started/astro) para inicializar a Lucia com um adaptador e implementar um banco de dados para armazenar usuários e sessões.
```